next/src/components/_global/nav/nav.tsx

```diff
@@ -82,20 +137,38 @@ export default function Nav({ data: { cta }, logo }: Props) {
         <div className={styles.hamburger}>{Hamburger({ toggleNav, setToggleNav })}</div>
       </div>
       {toggleNav && (
-        <div className={styles.toggled}>
+        <div
+          className={styles.toggled}
+          onMouseLeave={() => setToggleNav(!toggleNav)}
```

Comment:
Should be `onClick` event

Reply:
Why should it be onClick? I think it works perfectly fine as a onMouseLeave event